app/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>something</title>
```

Comment:
**TODO FOR LOST POINTS**

You need ``h1-h6's`` so you can use your ``scss`` iteration.